libs/labelbox/src/labelbox/data/serialization/ndjson/label.py

```diff
@@ -156,7 +162,7 @@ def _infer_media_type(
             raise ValueError("Missing annotations while inferring media type")
 
         types = {type(annotation) for annotation in annotations}
-        data = ImageData
+        data = GenericDataRowData
```

Comment:
Made a smalled change and made this GenericDataRowData this is for deserialization and eventually everything will be GenericDataRowData